test/TemperatureControlTest.cpp

```diff
@@ -0,0 +1,65 @@
+#include <Arduino.h>
+#include <ArduinoUnitTests.h>
+#include <ci/ObservableDataStream.h>
+
+#include "TankControllerLib.h"
+#include "TemperatureControl.h"
+
+/*
+bundle exec arduino_ci.rb --skip-examples-compilation --testfile-select=TemperatureControlTest.cpp
+*/
+
+//  void Chiller::updateControl(double currentTemperature);   // void TemperatureControl::turnOn(bool newState) {
```

Comment:
Don't include commented out code.

Reply:
I agree on removing (or explaining) this.